polaris.shopify.com/content/patterns/app-settings-layout.ts

```diff
@@ -23,7 +23,7 @@ const pattern: SingleVariantPattern = {
   |-|-|
   |Don't include a description unless it's helpful.|![](/images/patterns/app-settings-usage-1.png)|
   |Place grouped settings within cards.|![](/images/patterns/app-settings-usage-2.png)|
-  |Stack all setting groups vertically on the page.|![](/images/patterns/app-settings-usage-2.png)|`,
```

Comment:
this can be addressed in a followup PR:

Do we need alternative text for these images? if we do, add the alt text in between the square brackets
`![this is where alt text for images go in markdown](/images/patterns/app-settings-usage-2.png)`

if we don't need alt text on images you can leave this as is 👍